data/currencies.json

```diff
@@ -0,0 +1,34 @@
+{
+  "USD": "USD", "DOLLAR": "USD", "BUCK": "USD", "US DOLLAR": "USD",
```

Comment:
Can there be a more concise way of storing this information?
I don't like USD being repeated.